packages/web/app/src/components/v2/modals/create-access-token.tsx

```diff
@@ -329,18 +122,20 @@ function ModalContent(props: {
             project: props.projectId,
             target: props.targetId,
             name: values.name,
-            organizationScopes: filteredDownPermissions.organizationScopes,
-            projectScopes: filteredDownPermissions.projectScopes,
-            targetScopes: filteredDownPermissions.targetScopes,
+            organizationScopes: [],
+            projectScopes: [],
+            targetScopes:
+              selectedScope === 'no-access'
+                ? []
+                : selectedScope === TargetAccessScope.RegistryWrite
+                ? [TargetAccessScope.RegistryRead, TargetAccessScope.RegistryWrite]
```

Comment:
Does this show all ✅  when you are running `whoami` in the CLI?

Reply:
and maybe this should be in a separate function, just for clarity? something like `mapPermissions`? Because I think the complex condition is hard to read when it's inline

Reply:
```
  cli git:(feat-registry-access-token) ✗ pnpm start whoami

> @graphql-hive/cli@0.30.2 start /Users/laurinquast/Projects/graphql-hive-3/packages/libraries/cli
> ./bin/dev "whoami"

Token name:                    ddddd         
                                                      
Organization:                  brrrt         https://app.graphql-hive.com/brrrt
Project:                       gg            https://app.graphql-hive.com/brrrt/gg
Target:                        production    https://app.graphql-hive.com/brrrt/gg/production
                                                      
Access to schema:publish       Yes          
Access to schema:check         Yes          
Access to operation:publish    No access
```

Reply:
See https://github.com/kamilkisiela/graphql-hive/pull/3199/commits/c0f837a1904ea80bd4d970469e658d9ee206a4c2

Reply:
Perfect. Does it make sense not to have `operation:publish`? I mean, this token can't report usage to Hive?

Reply:
Hm it is the old persisted operations feature that we don't really use. 😅

Reply:
ah ok I see. Should we consider allowing it as well? I'm asking because you are working on the persisted operations feature, and it might be needed to have a token that can push operations/update the store?

Reply:
Since it is right now a project scope I am not really sure, as the one I am developing is target scoped 🤔